### PR TITLE
Iglu R9 Bull's Eye

### PR DESCRIPTION
**Pull request by oguzhanunlu**
@chuwy , this is an initial draft for blog post of R9 Bull's Eye.

### CODE REVIEW

**Comment by Mandelliant** [comment]
@chuwy @oguzhanunlu let me know when this is ready for posting

**Comment by alexanderdean** [comment]
Please add me as reviewer once @chuwy has finished...

**Comment by chuwy** [comment]
Hey @alexanderdean, I slightly restructured post and made some changes, but some of those changes imply that we've made decision on [this issue](https://github.com/snowplow/iglu/issues/131#issuecomment-378915756), if you're disagree with that - then I'll have to make another pass.

**Comment by alexanderdean** [comment]
Hey @oguzhanunlu - this looks great, nice work. A little bit verbose in places but this makes my job as editor much easier/faster (cutting down is 10x faster than building up).

If you have any questions about my edits, just ask them here. In a nutshell:

* Always lead on customer-facing improvements, not under-the-hood changes
* Always have an upgrading section. It forces you to put yourself in the shoes of an existing user wondering how to make use of the new version
* Always push users to the documentation for the latest version

**Comment by alexanderdean** [comment]
And thanks @chuwy for your work on this post too (lot of great restructuring there)!

**Comment by oguzhanunlu** [comment]
Hey @alexanderdean , thanks for the feedback. I'll fill in related sections and ask for final review.

**Comment by oguzhanunlu** [comment]
Hey @alexanderdean , there are few important upgrade notes that I added in latest commit. Could you take a look at them and review if they are too verbose etc. ?

File: _posts/2018-04-20-iglu-r9-released/index.md [hunk 1]
```diff
@@ -62,23 +62,21 @@ R9 Bull's Eye also fixes two important bugs in igluctl, introducing a new 0.4.1
...
 
-Note that protocol of URL should be omitted, i.e. omit `http(s)://`, since Swagger UI will automatically prepend that.
+Also, Iglu Server `0.3.0` negotiates `Content-Type` with clients. Clients should specify either `Accept: application/json` header or no `Accept` header at all. Resource representation is available only for `application/json`.
 
 Now you can launch server with following interface: `java -jar $JAR_PATH --config $CONFIG_PATH`.
 
 <h3 id="upgrade-igluctl">5.2 igluctl</h3>
 
-OGUZHAN TO EXPLAIN HOW TO UPDATE IGLUCTL
+igluctl `0.4.1` doesn't introduce any interface change. It is compatible with previous version `0.4.0`.
```
Comment (alexanderdean) [inline _posts/2018-04-20-iglu-r9-released/index.md]: Clarify how to download igluctl, the new version

File: _posts/2018-04-20-iglu-r9-released/index.md [hunk 2]
```diff
@@ -62,23 +62,21 @@ R9 Bull's Eye also fixes two important bugs in igluctl, introducing a new 0.4.1
 
 <h3 id="upgrade-iglu-server">5.1 Iglu Server</h3>
```
Comment (alexanderdean) [inline _posts/2018-04-20-iglu-r9-released/index.md]: Clarify how to download it and run it.

**Comment by alexanderdean** [comment]
Left some feedback, ping me again when you've updated

**Comment by oguzhanunlu** [comment]
ping @alexanderdean

**Comment by alexanderdean** [comment]
Looks good! I've made some final changes. On the content negotation - I hope what I changed made sense - if I've misunderstood what the content negotiation does, then revert that change and clarify what you mean.

**Comment by oguzhanunlu** [comment]
It's all good @alexanderdean , thanks!

**Pull request merged**